main/src/ca/uwaterloo/flix/language/phase/jvm/GenReifiedSourceLocationClass.scala

```diff
@@ -17,229 +17,170 @@
 package ca.uwaterloo.flix.language.phase.jvm
```

Comment:
Can you test that this generates correct code? I think if you trigger an exception then the source location should be printed.

Reply:
`Exception in thread "main" dev.flix.runtime.MatchError: Non-exhaustive match at examples\misc\test.flix:4:5`

Reply:
like some of the other cases i don't think equals/hashcode is used